### PR TITLE
[Task][68314] apply MVC structure

### PR DESCRIPTION
**Pull request by TuanKieuDev**
### [Redmine link]
https://edu-redmine.sun-asterisk.vn/issues/68314

#### Screenshots (nếu có)
Screen Shot 2023-09-12 at 10 21 43

### CODE REVIEW

File: MarvelApp/MarvelApp/Controller/ComicDetailViewController.swift [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+//  Created by Tobi on 12/09/2023.
+//
+
+import UIKit
+
+final class ComicDetailViewController: UIViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
```
Comment (hlinh88) [inline MarvelApp/MarvelApp/Controller/ComicDetailViewController.swift]: whitespace

File: MarvelApp/MarvelApp/Controller/ComicDetailViewController.swift [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+//  ComicDetailViewController.swift
+//  MarvelApp
+//
+//  Created by Tobi on 12/09/2023.
+//
+
+import UIKit
+
+final class ComicDetailViewController: UIViewController {
+
```
Comment (hlinh88) [inline MarvelApp/MarvelApp/Controller/ComicDetailViewController.swift]: whitespace

File: MarvelApp/MarvelApp/Controller/ComicViewController.swift [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+//  Created by Tobi on 12/09/2023.
+//
+
+import UIKit
+
+final class ComicViewController: UIViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
```
Comment (hlinh88) [inline MarvelApp/MarvelApp/Controller/ComicViewController.swift]:
```suggestion
    override func viewDidLoad() {
        super.viewDidLoad()
```

File: MarvelApp/MarvelApp/Controller/FavoriteViewController.swift [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+//  Created by Tobi on 12/09/2023.
+//
+
+import UIKit
+
+final class FavoriteViewController: UIViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
```
Comment (hlinh88) [inline MarvelApp/MarvelApp/Controller/FavoriteViewController.swift]:
```suggestion
    override func viewDidLoad() {
        super.viewDidLoad()
```

File: MarvelApp/MarvelApp/Controller/InformationDetailViewController.swift [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+//  Created by Tobi on 12/09/2023.
+//
+
+import UIKit
+
+final class InformationDetailViewController: UIViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
```
Comment (hlinh88) [inline MarvelApp/MarvelApp/Controller/InformationDetailViewController.swift]:
```suggestion
    override func viewDidLoad() {
        super.viewDidLoad()
```

File: MarvelApp/MarvelApp/Controller/SearchViewController.swift [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+//
+
+import UIKit
+
+final class SearchViewController: UIViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+    }
```
Comment (hlinh88) [inline MarvelApp/MarvelApp/Controller/SearchViewController.swift]:
```suggestion
    override func viewDidLoad() {
        super.viewDidLoad()
    }
```

**Review by hlinh88 — APPROVED** [review]

**Review by tunghs-1253 — APPROVED** [review]

**Pull request merged**